src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,58 @@
+.clock {
+  position: relative;
+
+  border: 5px dotted black;
+  border-radius: 50%;
+  background-color: grey;
+  height: $clock-diameter;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)